02-microarray/pathway-analysis_microarray_01_ora.Rmd

```diff
@@ -326,15 +345,15 @@ Over-representation testing using `clusterProfiler` is based on a hypergeometric
 
 Where `N` is the number of genes in the background distribution, `M` is the number of genes in a pathway, `n` is the number of genes we are interested in (our differentially expressed genes), and `k` is the number of genes that overlap between the pathway and our genes of interest.
```

Comment:
Readers may be familiar with a 2x2 contingency table, which could work as a complementary way to describe the data and may pair well with showing the formula (which may be a roadblock for some readers).

| | Genes of interest: | Yes | No | Total |
|-|-|-|-|-|
|Gene in pathway: | Yes | k | M - k | M |
| |No | n - k | N - M + n + k | N - M |
| | Total | n | N - n | N |

Reply:
I like this idea but its a bit outside the scope of this PR. But, I think this change should happen when we address the other ORA items with #306 So I will track this idea there for when I get to that.